pylibfuzzer/exec/runner.py

```diff
@@ -157,6 +170,10 @@ def run(self):
                             f.write(data)
                     results.append(self.pipeline.batch_transform(result))
 
+                    if self.create_dataset:
+                        dataset_in.write(data + b'\n')
```

Comment:
i think its not good if we ad a newline character to the file. i think it should not be altered in any way.